app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class UserType < Types::BaseObject
+    field :id, ID
+    field :first_name, String
+    field :last_name, String
+    field :email, String
+    field :role, String
```

Comment:
It is okay, but ideally it should be special type:
```ruby
module Types
  class UserRoleEnum < BaseEnum
    User.role.each_value do |user_role|
      value user_role.to_s.upcase, user_role.text, value: user_role.to_s
    end
  end
end
```

Reply:
Thanks